src/components/application_manager/test/commands/hmi/dummy_hmi_commands_test.cc

```diff
@@ -0,0 +1,546 @@
+#include "application_manager/commands/hmi/on_vi_prndl_notification.h"
+#include "application_manager/commands/hmi/on_vi_tire_pressure_notification.h"
+#include "application_manager/commands/hmi/on_vi_odometer_notification.h"
+#include "application_manager/commands/hmi/on_vi_belt_status_notification.h"
+#include "application_manager/commands/hmi/on_vi_body_information_notification.h"
+#include "application_manager/commands/hmi/on_vi_device_status_notification.h"
+#include "application_manager/commands/hmi/on_vi_driver_braking_notification.h"
+#include "application_manager/commands/hmi/on_vi_wiper_status_notification.h"
+#include "application_manager/commands/hmi/on_vi_head_lamp_status_notification.h"
+#include "application_manager/commands/hmi/on_vi_engine_torque_notification.h"
+#include "application_manager/commands/hmi/on_vi_acc_pedal_position_notification.h"
+#include "application_manager/commands/hmi/on_vi_steering_wheel_angle_notification.h"
+#include "application_manager/commands/hmi/on_vi_my_key_notification.h"
+#include "application_manager/commands/hmi/ui_set_icon_request.h"
+
+#include "application_manager/mock_application.h"
+#include "application_manager/mock_application_manager.h"
+#include "test/application_manager/mock_application_manager_settings.h"
+#include "application_manager/mock_message_helper.h"
+#include "application_manager/mock_event_dispatcher.h"
+
+namespace am = application_manager;
+
+namespace test {
+namespace components {
+namespace commands_test {
+namespace hmi_commands_test {
+
+namespace commands = ::application_manager::commands;
+
+using ::testing::_;
+using ::testing::NotNull;
+using ::testing::Types;
+using commands::MessageSharedPtr;
+using ::test::components::event_engine_test::MockEventDispatcher;
+using ::test::components::application_manager_test::MockApplicationManager;
+using ::test::components::application_manager_test::
+    MockApplicationManagerSettings;
+using ::application_manager::ApplicationSharedPtr;
+using ::test::components::application_manager_test::MockApplication;
+using ::application_manager::MockMessageHelper;
+
+template <class Command>
+class HMICommandsTest : public CommandsTest<CommandsTestMocks::kIsNice> {
+ public:
+  typedef Command CommandType;
+
+  void InitCommand(const uint32_t& timeout) OVERRIDE {
+    stream_retry_.first = 0;
+    stream_retry_.second = 0;
+    EXPECT_CALL(app_mngr_set_, default_timeout()).WillOnce(ReturnRef(timeout));
+    ON_CALL(app_mngr_, event_dispatcher())
+        .WillByDefault(ReturnRef(mock_event_dispatcher_));
+    ON_CALL(app_mngr_, get_settings()).WillByDefault(ReturnRef(app_mngr_set_));
+    ON_CALL(app_mngr_set_, start_stream_retry_amount())
+        .WillByDefault(ReturnRef(stream_retry_));
+  }
+
+ protected:
+  std::pair<uint32_t, int32_t> stream_retry_;
+  NiceMock<MockApplicationManagerSettings> app_mngr_set_;
+  NiceMock<MockEventDispatcher> mock_event_dispatcher_;
```

Comment:
@okozlovlux, `MockEventDispatcher` already included in `CommandRequestTest` class. Use it to deal with request commands.